### PR TITLE
upgrade some dependencies

### PR DESCRIPTION
**Pull request by aliceriot**
this will let us resolve some security alerts in other projects I think

make sure tests pass, changes look good, things build, etc.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,13 +29,13 @@
...
+    "babel-runtime": "^6.26.0",
     "isomorphic-fetch": "^2.2.1",
-    "lodash.merge": "^4.6.0",
+    "lodash.camelcase": "^4.3.0",
+    "lodash.merge": "^4.6.2",
     "lodash.snakecase": "^4.1.1",
-    "lodash.camelcase": "^4.1.1",
-    "ramda": "^0.24.1",
-    "redux-actions": "^2.0.3"
+    "ramda": "0.24.1",
```
Comment (noisecapella) [inline package.json]: Should we try updating this too? Or too much for this PR?

Comment (aliceriot) [reply package.json]: I updated it and it broke stuff so I said nahhh, haha

Comment (aliceriot) [reply package.json]: but we should in the future. i'll file an issue here so we keep track of it.

Comment (aliceriot) [reply package.json]: filed #29

**Pull request merged**